packages/notifi-dapp-example/src/components/ConfigDestinationModal.tsx

```diff
@@ -29,7 +30,7 @@ export const ConfigDestinationModal: React.FC<ConfigDestinationModalProps> = ({
           </p>
         </div>
         <div className="flex flex-col items-center justify-center">
-          {loading ? (
+          {isGlobalLoading ? (
```

Comment:
Should we remove the LoadingSpinner here? as we already have the loadingSpinner when isGlobalLoading is true